microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Understand the heap snapshot file format
+description: How the heap snapshot files are structured, when exported from the Memory tool
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 9/16/2022
+---
+# Understand the heap snapshot file format
+
+Investigating memory usage in web applications can be hard. The DevTools **Memory** tool allows you to explore all the objects allocated in memory by your web application by taking a heap snapshot. This information is useful for performance investigations because you can find out what objects are consuming the most memory.
```

Comment:
```suggestion
Investigating memory usage in web applications can be difficult. The DevTools **Memory** tool allows you to explore all the objects that are allocated in memory by your web application, by taking a heap snapshot. This information is useful for performance investigations, because you can find out which objects are consuming the most memory.
```